.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,52 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: "Chromatic"
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    # Operating System
+    runs-on: ubuntu-latest
+    # Job steps
+    steps:
+      - name: Get Yarn cache path
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Checkout
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+
+      - name: Enable node
+        uses: actions/setup-node@master
+        with:
+          node-version: 12.x
+
+      - name: Load Yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile && yarn bootstrap
+
+      - name: Build packages
+        run: yarn build
+
+        # 👇 Adds Chromatic as a step in the workflow
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        # Chromatic GitHub Action options
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
```

Comment:
do checkout the following options in the docs:
```
onlyChanged: true
exitOnceUploaded: true
```

first one will run visual diffing tests only on changed components. since chromatic limits visual diff tests for free projects, we don't want to run those if components haven't changed

`exitOnceUploaded: true` will terminate the action once components are sent to chromatic and will save compute time used by github actions

read here: [chromatic docs](https://www.chromatic.com/docs/github-actions#available-options)

Reply:
@burhanuday there is no `onlyChanged` option in the docs.
Are you talking about `exitZeroOnChanges`? It's `true` by default.

Reply:
@isBatak I was referring to their TurboSnap feature here - https://www.chromatic.com/docs/turbosnap#enable

Reply:
I guess we need to update Storybook to v6.2 then

Reply:
Also in this phase I'm concerned with heuristic of change detection and false positives 
https://www.chromatic.com/docs/turbosnap#how-it-works

Reply:
Hmm. I have been using chromatic with this enabled for a couple of months. I haven't faced the issue. But in any case, let's not enable it if they don't recommend doing so